### PR TITLE
Motor -> core

### PR DESCRIPTION
**Pull request by YHerm**

### CODE REVIEW

File: src/main/java/frc/robot/hardware/motor/ControlAble.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package frc.robot.hardware.motor;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.request.IRequest;
+import frc.utils.calibration.sysid.SysIdCalibrator;
+
+public interface ControlAble {
```
Comment (YoniKiriaty) [inline src/main/java/frc/robot/hardware/motor/ControlAble.java:7]: Why the capitalization??
Also controllable is with two Ls

**Review by NoamRosenberg08 — APPROVED** [review]

**Review by hillelv — DISMISSED** [review]

File: src/main/java/frc/robot/hardware/motor/phoenix6/TalonFXMotor.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import frc.robot.hardware.motor.Controllable;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.robot.hardware.motor.IMotor;
+import frc.robot.hardware.phoenix6.Phoenix6Device;
+import frc.robot.hardware.request.IRequest;
+import frc.robot.hardware.request.phoenix6.Phoenix6AngleRequest;
+import frc.robot.hardware.request.phoenix6.Phoenix6DoubleRequest;
+import frc.utils.calibration.sysid.SysIdCalibrator;
+
+public class TalonFXMotor extends Phoenix6Device implements IMotor, Controllable, ControllableMotor {
```
Comment (tnrz135) [inline src/main/java/frc/robot/hardware/motor/phoenix6/TalonFXMotor.java:15]: ControllableMotor already gives IMotor and Controllable, no need to write twice

**Review by hillelv — APPROVED** [review]

**Review by YoniKiriaty — DISMISSED** [review]

**Review by tnrz135 — APPROVED** [review]

**Review by YHerm — COMMENTED** [review]
I can't approve it because i opened it, WE NEED TO MAKE SURE IT IS ALL TESTED AFTER THE CR CHANGES

**Comment by drorta** [comment]
> I can't approve it because i opened it, WE NEED TO MAKE SURE IT IS ALL TESTED AFTER THE CR CHANGES

So we test it on Wednesday?

**Review by NoamRosenberg08 — DISMISSED** [review]

**Review by NoamRosenberg08 — APPROVED** [review]

**Review by drorta — DISMISSED** [review]

**Review by drorta — APPROVED** [review]

**Pull request merged**